src/commands/createWebApp/setAppWizardContextDefault.ts

```diff
@@ -72,3 +68,14 @@ export async function setAppWizardContextDefault(wizardContext: IAppServiceWizar
         }
     }
 }
+
+export function resetAppWizardContextDefaults(context: Partial<IAppServiceWizardContext>): void {
+    context.plan = undefined;
```

Comment:
I wonder if instead of individually setting these to undefined (which we would have to maintain going forward if we add more properties), you could just create a clone of `context` before calling `setAppWizardContextDefault` and then use that for advanced create